VargasMartinezJose/Dis45/HackerRank/ejerciciojava.java/JavaIfElse.java

```diff
@@ -0,0 +1,33 @@
+import java.util.*;
```

Comment:
No using wildcards in your imports.
The default unnamed package shouldn't be used.